charter.md

```diff
@@ -0,0 +1,34 @@
+# Style Team
+
+## Aims: Evolving the Rust Style over time
+
+1. Selecting styling for new Rust constructs
+1. Evolving the existing style over the course of Rust editions (without breaking backwards compatibility)
+1. Defining mechanisms to evolve the Rust style while taking backwards compatibility into account, such as via Rust editions or similar mechanisms
+
+## Domains
+
+1. [Style Guide]
+1. [T-style team repo](https://github.com/rust-lang/fmt-rfcs/)
+1. [#T-style] and #T-style/private zulip streams
+1. `rust-lang` issues with the `T-style` or `T-style-nominated` labels
+
+## Membership
+
+* Caleb Cartwright ([@calebcartwright](https://github.com/calebcartwright))
+* Jane Losare-Lusby ([@yaahc](https://github.com/yaahc))
+* Josh Triplett ([@joshtriplett](https://github.com/joshtriplett))
+* Michael Goulet ([@compiler-errors](https://github.com/compiler-errors))
+
+## Membership
```

Comment:
did you intend to have this title twice?

Reply:
no, good catch ^_^